Package.swift

```diff
@@ -3,24 +3,24 @@
 import PackageDescription
 
 let package = Package(
-    name: "KommunicateCore-iOS-SDK",
+    name: "KommunicateCore_iOS_SDK",
```

Comment:
We are only changing the name in SPM, not in CP?

Reply:
yes